code/datums/action.dm

```diff
@@ -277,11 +277,22 @@
 	icon_icon = 'icons/mob/actions/actions_spacesuit.dmi'
 	button_icon_state = "thermal_off"
 
+/datum/action/item_action/toggle_spacesuit/New(Target)
```

Comment:
this should be in Initialize, as we prefer to avoid New for most uses now.

and you will want to `return ..()` in Intialize

Reply:
code\datums\action.dm:280:error: Initialize: undefined proc

Reply:
Apparently Initialize is an atom level proc, just ignore me